radicle-cli/src/commands/assign.rs

```diff
@@ -70,9 +67,9 @@ impl Args for Options {
 
         Ok((
             Options {
-                id: id.ok_or_else(|| anyhow!("an issue must be specified"))?,
+                id: id.ok_or_else(|| anyhow!("an issue ID must be specified"))?,
                 to: NonEmpty::from_vec(to)
-                    .ok_or_else(|| anyhow!("an assignee must be specified"))?,
+                    .ok_or_else(|| anyhow!("an assignee DID must be specified"))?,
```

Comment:
I wouldn't touch these.

Reply:
It's helpful to know what kind of type the command expects, but I can revert.